e2e/schematics/command-line.test.ts

```diff
@@ -175,7 +175,7 @@ describe('Command line', () => {
       );
 
       const output = runCommand(
-        'npm run workspace-schematic -- custom mylib --directory=dir'
+        'npm run workspace-schematic custom mylib -- --directory=dir'
```

Comment:
I think some of your other changes snuck in here

Reply:
@ThomasBurleson do you know how this works? I thought everything after -- gets passed to your script, and everything before -- gets consumed by npm itself.